pkg/private/zip/build_zip.py

```diff
@@ -125,12 +217,14 @@ def main(args):
   if args.mode:
     default_mode = int(args.mode, 8)
 
-  with zipfile.ZipFile(args.output, mode='w') as zip_file:
-    if args.manifest:
-      with open(args.manifest, 'r') as manifest_fp:
-        manifest = json.load(manifest_fp)
-        for entry in manifest:
-          _add_manifest_entry(args, zip_file, entry, default_mode, ts)
+  if not args.manifest:
+    raise Exception('Missing --manifest')
```

Comment:
Maybe add `required=True` to the declaration of this argument in `_create_argument_parser`?

Reply:
Yes. That makes sense. Done